File: homework_5/calculations/Minus.java
package calculations;

public class Minus implements ICalculator {
    private double a;
    private double b;

    public Minus(double a, double b){
        this.a = a;
        this.b = b;
    }

    @Override
    public double getResult(){
        return a - b;
    }
}

File: homework_7/calculations/minus.py
from calculations.I__calculator import Calculations

"""наследуем класс Minus от базового Calculations"""
class Minus(Calculations):
    def __init__(self, a, b):
        self.a = a
        self.b = b
    
    """переопределяем абстрактый метод"""
    def get_result(self):
        return self.a - self.b